frontend/js/src/recent/RecentListens.tsx

```diff
@@ -55,13 +57,16 @@ export default class RecentListens extends React.Component<
                 <small className="text-muted">songs played</small>
               </div>
             </Card>
-            <Card id="listen-count-card">
+            <Card id="listen-count-card" className="card-user-sn">
               <div>
                 {globalUserCount ?? "-"}
                 <br />
                 <small className="text-muted">users</small>
               </div>
             </Card>
+            <Card className="hidden-xs hidden-sm">
```

Comment:
Not sure I like that it is hidden on some screen sizes, but I suppose if that's what we do with other cards there, we should be coming up with a solution for the other ones too.
Maybe not necessary for sm sizes?
`<Card className="hidden-xs">`